express_relay/sdk/js/src/examples/simpleSearcherLimo.ts

```diff
@@ -112,41 +119,49 @@ class SimpleSearcherLimo {
       order.address,
       bidAmount,
       new anchor.BN(Math.round(Date.now() / 1000 + DAY_IN_SECONDS)),
-      this.chainId
+      this.chainId,
+      this.expressRelayConfig!.relayerSigner,
+      this.expressRelayConfig!.feeReceiverRelayer
     );
 
+    bid.transaction.recentBlockhash = opportunity.blockHash;
+    bid.transaction.sign(this.searcher);
+    return bid;
+  }
+
+  async opportunityHandler(opportunity: Opportunity) {
```

Comment:
Can we change the code so that opportunityHandler get a svm opportunity as input?

Reply:
I don't think so, this would affect all the subscription interfaces.